src/Plugin/tmgmt/Translator/PermissionTranslator.php

```diff
@@ -525,6 +525,7 @@ public function localTaskItemBreadcrumbAlter(Breadcrumb &$breadcrumb, RouteMatch
     /** @var \Drupal\tmgmt_local\LocalTaskItemInterface $local_task_item */
     $local_task_item = $route_match->getParameter('tmgmt_local_task_item');
     $job_item = $local_task_item->getJobItem();
+    $item_type = $job_item->getItemType();
```

Comment:
We could do this inside the try or just before it so in case the method returns early we don't set it for nothing.